src/util/generateConnection.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from './fs';
+import { Key, Logger, Mapping } from './types';
+
+const CONNECTIONS_FILE_NAME = 'connections.sync-db.json';
+const KEYS = ['DB_HOST', 'DB_PASSWORD', 'DB_NAME', 'DB_USERNAME', 'DB_PORT', 'DB_CLIENT', 'DB_ENCRYPTION'];
+const mapping: Mapping<string> = {
+  USERNAME: 'name',
+  NAME: 'database'
+};
+
+/**
+ * Reads values from ENV from provide keys.
+ *
+ * @param {string[]} keys
+ * @returns {string[] | Key}
+ */
+function extractFromEnv(keys: string[]): string[] | Key {
```

Comment:
Should return a list of database connections (only has one item for now).
```suggestion
function extractFromEnv(keys: string[]): ConnectionConfig[] {
```